VCEntities/VCEntities/backup/content/microsoft2020/RawIdentity.swift

```diff
@@ -65,18 +65,18 @@ struct RawIdentity: Codable {
                           alias: self.name)
     }
     
-    func jwkFromKeyContainer(_ keyContainer: KeyContainer) throws -> Jwk {
+    func jwkFromKeyContainer(_ keyContainer: KeyContainer) throws -> JWK {
 
         // Get out the private and public components of the key (pair)
         let secret = keyContainer.keyReference
-        let (privateKey, publicKey) = try Secp256k1().createKeyPair(forSecret: secret)
+        let (privateKey, publicKey) = try Secp256k1(secret: secret).createKeyPair(forSecret: secret)
```

Comment:
Why do we need to pass the secret in both the constructor and method?

Reply:
reverted.